integration-tests/rpc-get-protocol-constants.spec.ts

```diff
@@ -598,7 +598,7 @@ CONFIGS().forEach(({ lib, protocol, rpc }) => {
 
   describe(`Fetch constants for testnet`, () => {
 
-    mumbainet(`successfully fetches all constants for mumbainet using ${rpc}`, async (done) => {
+    mumbaiPlus(`successfully fetches all constants for mumbainet using ${rpc}`, async (done) => {
```

Comment:
I think we should instead pinpoint these to certain networks. I.e. have them be only nairobinet, mumbainet, etc.

It would make it easier for us to determine what to remove/add in the future since this test file will evolve overtime

Reply:
I think pinning to a specific version is dangerous. By default, these are expected to run for all networks after the specific network. If it fails, we will notice and investigate. But if it gets skipped, it can stay unnoticed.

Later, we can simply change it to `it` or `test`

Reply:
Updated